### ISSUE TITLE
shadowsocks server doesnt work expectly.

### ISSUE THREAD

**Issue by goodarzif** [jlund/streisand#298]
i have installed shadowsocks on vps server. everything seems to work correctly. when i start shadowsocks on server, it outputs:
initialize ciphers... bf-cfb
 listening at blablabla
so it runs correctly.
and on client when i use firefox with the proper config for shadowsocks. i see it connect to my vps, but no data transfer occur.
the configuration of both client and server is equal.
thanks in advance for any help.

**Comment by hydrandt** [comment]
From what you write it seems that you are trying to connect your Firefox directly to the shadowsocks server. Did you start the shadowsocks client first on your local machine? Then you have to connect your browser to your local shadowsocks instance.

**Comment by goodarzif** [comment]
yes, of course. i have used below to my client, and when go to an url address in firefox with proper config, I see:
```
sslocal -c /etc/shadowsocks/config.json -v
INFO: loading config from /etc/shadowsocks/config.json
2016-05-23 16:26:00 INFO     loading libcrypto from libcrypto.so.10
2016-05-23 16:26:00 INFO     starting local at 127.0.0.1:2465
2016-05-23 16:26:00 DEBUG    using event model: epoll
2016-05-23 16:26:05 DEBUG    accept
2016-05-23 16:26:05 DEBUG    chosen server: My IP Address:port 
2016-05-23 16:36:06 WARNING  timed out
2016-05-23 16:36:06 DEBUG    destroy
2016-05-23 16:36:06 DEBUG    destroying local
```
and you can see after 10 minutes it says `WARNING timed out`.

and this is my config.json file:
```
{
    "server":"my ip address",
    "server_port":blabla,
    "local_port":2465,
    "password":"blabla",
    "timeout":600,
    "method":"bf-cfb",
    "auth": false
}
```
another thing:
Is this for chown or chmod?if yes. which file?

**Comment by jlund** [comment]
Something seems to be off here. Streisand's Shadowsocks setup doesn't use Blowfish. Did you make any changes to the configuration?

**Comment by goodarzif** [comment]
first, sorry to comment late. 
no, i dont. i just create config.json file on both side, that can you see above.
on the server, i use ss-server command. but on client, i use sslocal ( without dash, between).
Is it for this?
I have used linux-ubuntu on server and linux-fedora on client. On both side, does shadowsocks use someplace to write something. maybe I should give some access to shadowsocks on that place. (via chmod and chown)

**Comment by jlund** [comment]
You shouldn't be having to create the configuration file on the server. That happens automatically. Streisand also sets the proper permissions on all of the files too.

I can't seem to reproduce this. Shadowsocks is working as expected on a fresh server using the provided configuration and setup instructions.
